.github/renovate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "local>hmcts/.github:renovate-config"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "helmv3": {
+    "bumpVersion": "patch"
+  },
+  "regexManagers": [
+    {
+      "fileMatch": [
+        "^Dockerfile$"
+      ],
+      "matchStrings": [
+        "datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\sARG .*?_VERSION=(?<currentValue>.*)\\s"
+      ]
+    }
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": [
+        "*"
+      ],
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
+      ],
+      "groupName": "all non-major dependencies",
+      "groupSlug": "all-minor-patch",
+      "automerge": true
+    }
+  ],
+  "prConcurrentLimit": 10
```

Comment:
Renovate will try to open all PRs at the same time for all repositories, producing a massive spike in the number of running builds. We're not overly pushed with having all the updates at the same time.

Reduce this number to 3 and make it do in a daily schedule, at a time of your choice (for other repositories we'll make it happen another time). This will help keeping the number of PRs low. If the automerge works, this won't be a problem. If it doesn't (let's say because the build failed) then it's up to developers to sort out the problem.